api/src/routes/api/mock-sas9.ts

```diff
@@ -0,0 +1,145 @@
+import { readFile } from '@sasjs/utils'
+import express from 'express'
+import path from 'path'
+
+const mockSas9Router = express.Router()
+
+const { MOCK_SERVERTYPE } = process.env
+
+let loggedIn: boolean = false
+
+mockSas9Router.get('/SASStoredProcess', async (req, res) => {
+  if (!loggedIn) {
+    res.redirect('/SASLogon/login')
+    return
+  }
+
+  const filePath = path.join(
+    process.cwd(),
+    'mocks',
+    'generic',
+    'sas9',
+    'sas-stored-process'
+  )
+
+  let file
+
+  try {
+    file = await readFile(filePath)
+  } catch (err: any) {
+    console.error(`Mocked file on path: ${filePath} is not found.`)
+    res.status(403).send(err.toString())
+    return
+  }
+
+  try {
+    res.send(file)
+  } catch (err: any) {
+    res.status(403).send(err.toString())
+  }
+})
+
+mockSas9Router.post('/SASStoredProcess/do/', async (req, res) => {
+  let program = req.query._program?.toString() || ''
+  program = program.replace('/', '')
+  const filePath = path.join(process.cwd(), 'mocks', ...program.split('/'))
+
+  let file
+
+  try {
+    file = await readFile(filePath)
+  } catch (err: any) {
+    let err = `Mocked file on path: ${filePath} is not found.`
```

Comment:
How can you explicitly say that file is not found? It can be any other error in the file reading. And again I would say that we should use .then .catch approach instead of try catch.